torch/ao/quantization/fx/quantization_patterns.py

```diff
@@ -243,6 +243,8 @@ def convert(self,
     torch.nn.functional.softmax: int8_dtypes,
     torch.nn.functional.dropout: int8_dtypes,
     torch.sum: fp16_dtypes,
+    torch.nn.PReLU: int8_dtypes,
+    torch.nn.functional.prelu: int8_dtypes,
```

Comment:
this is deprecated I think, we have some refactoring going on right now and new ops need to be added in torch/ao/quantization/fx/backend_config/native.py